libbeat/processors/add_host_metadata/config.go

```diff
@@ -34,10 +35,24 @@ type Config struct {
 }
 
 func defaultConfig() Config {
-	return Config{
-		NetInfoEnabled:      true,
-		CacheTTL:            5 * time.Minute,
-		ExpireUpdateTimeout: time.Second * 10,
-		ReplaceFields:       true,
+	// Setting environmental variable NETINFO:false in Elastic Agent pod will disable the netinfo.enabled option of add_host_metadata processor
+	// This will result to events not being enhanced with host.ip and host.mac
+	// Related to https://github.com/elastic/integrations/issues/6674
+	valueNETINFO, _ := os.LookupEnv("NETINFO")
```

Comment:
My only question would be the naming we want to choose here. Maybe sth like `ELASTIC_NETINFO` would be more explicit?

Reply:
I agree lets not introduce a breaking change (I was thinking an old comment you had done). 

For the naming you are killing me :) but I was expecting the debate , so seeing the page again https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html, I would go for ELASTIC_AGENT_NETINFO as strickly speaking this is a change on the agent side. 

I would wait until tomorrow to see if other have any opinion and I would need to change all prs